cishouseholds/pipeline/validation_calls.py

```diff
@@ -23,135 +30,130 @@ def validation_calls(SparkVal):
                 "allow_none": True,
             }
         },
-        "visit_id": {"starts_with": r"DHV"},
-        "blood_sample_barcode": {
-            "matches": r"^(ON([SWCN]0|S2|S7)[0-9]{7})$",
-            "subset": F.col("survey_response_dataset_major_version") <= 2,
-        },
-        "swab_sample_barcode": {
-            "matches": r"^(ON([SWCN]0|S2|S7)[0-9]{7})$",
-            "subset": F.col("survey_response_dataset_major_version") <= 2,
-        },
-    }
-    duplicate_column_calls = {
-        "blood_sample_barcode": {
-            "matches": r"^BLT[0-9]{8}$",
-            "subset": F.col("survey_response_dataset_major_version") == 3,
-        },
-        "swab_sample_barcode": {
-            "matches": r"^SWT[0-9]{8}$",
-            "subset": F.col("survey_response_dataset_major_version") == 3,
-        },
+        # "blood_sample_barcode": {
+        #     "matches": r"^BLT[0-9]{8}$",
+        #     "subset": F.col("survey_response_dataset_major_version") == 3,
+        # },
+        # "swab_sample_barcode": {
+        #     "matches": r"^SWT[0-9]{8}$",
+        #     "subset": F.col("survey_response_dataset_major_version") == 3,
+        # },
     }
+
     for col in SparkVal.dataframe.columns:
         if col in category_maps["iqvia_raw_category_map"]:
             column_calls[col] = {"isin": list(category_maps["iqvia_raw_category_map"][col].keys())}
 
     SparkVal.validate_column(column_calls)
-    SparkVal.validate_column(duplicate_column_calls)
 
     vaccine_columns = []
-    for template in ["cis_covid_vaccine_type_{}", "cis_covid_vaccine_type_other_{}", "cis_covid_vaccine_date_{}"]:
+    for template in ["covid_vaccine_type_{}", "covid_vaccine_type_other_{}", "covid_vaccine_date_{}"]:
```

Comment:
even though the check has been removed, can we change this back to cis_covid_vaccine for when/if the check gets added back in?

Reply:
Will do - apologies, thought I had done this already.